getting_started/step_by_step/exporting.rst

```diff
@@ -119,6 +119,13 @@ changed:
         emit_signal("hit")
         $CollisionShape2D.set_deferred("disabled", true)
 
+Setting a main scene
+--------------------
+
+The main scene is the one that your game will start in. In *Project -> Project 
+Settings -> Application -> Run*, set *Main Scene* to "Main.tscn" by clicking 
```

Comment:
Being more explicit about "Main.tscn" might be helpful? Is this a convention that should be followed or is it just an example? Might be worth covering main scene naming conventions (if it's worth it) in https://github.com/godotengine/godot-docs/issues/2997?

Reply:
Code architecture (where you place your classes, how your organize your files, separate systems) and naming might be beyond the scope of _GDScript_ programming guidelines. Your main scene could be anything, like the entry point of a python program can be any function, module, or package, really.